app/views/posts/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<h4>New Post</h4>
+
+<%= form_for @post do |f| %>
+
+  <div class="form-group">
+    <label>Title</label>
+    <%= f.text_field :title, class: "form-control" %>
+    <label>Content</label>
+    <%= f.text_field :content, class: "form-control" %>
+    <label>ID User</label>
+    <%= f.text_field :user_id, class: "form-control" %>
+    <% @post.user = user_session %>
```

Comment:
There are a lot of ways to do this, but this one is not a good one. Instead of sending the `user_id` and getting the user from `user_session` you can do in the controller:
```
@post = current_user.posts.new(post_params)
```